examples_gbs/run_tutorial_points.py

```diff
@@ -0,0 +1,137 @@
+
+##############################################################################
+# The rows of R are the coordinates of the points.
+#
+# Next step is to create the kernel matrix for the points of this discrete space. We call
+# the :func:`~.kernel` function which uses the *radial basis function* (RBF) kernel defined as:
+#
+# .. math::
+#     K_{i,j} = e^{-\|\bf{r}_i-\bf{r}_j\|^2/2\sigma^2},
+#
+# where :math:`\bf{r}_i` are the coordinates of point :math:`i` and :math:`\sigma` is a kernel
+# parameter that determines the scale of the kernel.
+#
+# In the RBF kernel, points that are much further than a distance :math:`\sigma` from each other
+# lead to small entries of the kernel matrix, whereas points much closer than :math:`\sigma`
+# generate large entries. Remember that the permanent of a matrix is a sum over the product of
+# some matrix entries. Now consider a specific point pattern in which all points are close to each
+# other which simply means that their matrix elements have larger entries. Therefore, the
+# submatrix that corresponds to those points has a large permanent and the probability of
+# observing them in a sample is larger.
+#
+# For kernel matrices that are positive-semidefinite, such as the RBF kernel, there exist efficient
+# quantum-inspired classical algorithms for permanental point process sampling
+# :cite:`jahangiri2019point`. In this tutorial we use positive-semidefinite
+# kernels and the quantum-inspired classical algorithm.
+#
+# Let's construct the RBF kernel with the parameter :math:`\sigma` set to 1.0.
+
+K = points.kernel(R, 1.0)
+
+##############################################################################
+# We generate 10 samples with an average number of 40 points per sample by calling
+# the :func:`~.points.sample` function of the :mod:`~.gbs.points` module.
+
+samples = points.sample(K, 40.0, 10)
+
+##############################################################################
+# We visualize the first sample by using the :func:`~.plot_points` function of
+# the :mod:`~.gbs.plot` module. The point patterns generated by the permanental point process
+# typically have a higher degree of clustering compared to a uniformly random pattern.
+
+plotly.offline.plot(
+    plot.plot_points(R, samples[0], point_size=10), filename="Points.html"
+)
+
+##############################################################################
+# .. raw:: html
+#     :file: ../../examples_gbs/Points.html
+
+##############################################################################
+# Outlier Detection
+# -----------------
+#
+# When the distribution of points in a given space is inhomogeneous, GBS point processes typically
+# sample points from the dense regions. In a data set containing a uniform grid of points and
+# some dense clusters, this feature of the GBS point processes can be used to detect the
+# high-density points that can be considered as outliers with respect to the homogeneous
```

Comment:
Is this a standard notion for an outlier? I would think that the more usual sense of the word is that an outlier is a point that doesn't belong to a cluster...

Reply:
I have changed the definition: the clusters are the normal points and the outliers are a minority of points that belong to a random noisy background.